tests/test_geojson.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 from pathlib import Path
 
-from urbanopt_des.urbanopt_geojson import URBANoptGeoJSON
+from urbanopt_des.urbanopt_geojson import DESGeoJSON
```

Comment:
I renamed the uo-des geojson parsing class, to not conflict with the one from the GMT. Should it be different?Happy to change the name to something else if that makes more sense.

Reply:
I think we should move all the methods out of DESGeoJSON... looks like the only one remaining is `create_aggregated_representation`. Can we move that over to GMT's GeoJSON?